corehq/form_processor/migrations/0035_remove_varchar_pattern_ops_indexes.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import uuidfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('form_processor', '0034_location_id_index'),
+    ]
+
+    NOOP_REVERSE = 'SELECT 1'
+    operations = [
+        migrations.RunSQL(
+            'DROP INDEX IF EXISTS form_processor_xforminstancesql_form_uuid_12662b9ceadeeecc_like',
```

Comment:
They were the same on my machine, prod and staging. Could it be related to
using a very different version of postgres?